content/docs/start/data-pipelines.md

```diff
@@ -7,7 +13,9 @@ that produce a final result.
 
 DVC pipelines and their data can also be easily versioned (using Git). This
 allows you to better organize your project, and reproduce your workflow and
-results later exactly as they were built originally!
+results later exactly as they were built originally! This allows you to capture
+simple ETL workflows, better organize data science projects, or build detailed
+machine learning pipelines (to name a few uses).
```

Comment:
Actually this text has a couple phrasing issues:
- DVC pipelines... allows you to... build detailed machine learning pipelines? Idk, seems like a circular explanation
- "This allows you to" is repeated.

Reply:
OK, edited this to fix phrasing while avoiding too much copy editing. PTAL

Reply:
Looks good 👍